### DISCUSSION TITLE
导出显示异常

### DISCUSSION THREAD

**Discussion by HRS2049** [tamlok/vnote#2008] [Q&A]
请教：用wkhtmltopdf 和pandoc导出为pdf及epub时，出现如下问题，请问：该如何设置才能正常显示？谢谢！
![image](https://user-images.githubusercontent.com/96814608/147877270-20e37f38-715b-4cbb-a10f-836363cff7f4.png)

**Comment by tamlok** [comment]
导出pdf的问题我看一下。导出epub的问题我就不看了，你可以试试先导出html，然后自己用pandoc转换成epub看看是不是pandoc本身的问题。如果要kindle看的话，可以尝试导出为docx，然后发送到kindle，或者导出为6寸的PDF。

**Reply by HRS2049** [reply]
谢谢大神。

**Reply by HRS2049** [reply]
已经初步确认为pandoc的问题。但是用calibre可以完美将html转换为epub。

**Comment by tamlok** [comment]
vnote自定义导出的源格式可以选择html，然后调用calbre来直接导出，如果后者有命令行提供的话。

**Reply by HRS2049** [reply]
用VNOTE先导出为html网页，再用calibre加载html，然后点击转换为EPUB就完事了。再次感谢这个非常棒的笔记软件。
祝好!
